cinema/migrations/0002_movie_image.py

```diff
@@ -0,0 +1,21 @@
+# Generated by Django 4.1 on 2024-11-29 09:11
+
+import cinema.models
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ("cinema", "0001_initial"),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name="movie",
+            name="image",
+            field=models.ImageField(
+                null=True, upload_to=cinema.models.create_custom_path
```

Comment:
Ensure that the `create_custom_path` function in `cinema.models` is correctly defined to format the image path as `f"{slugify(movie.title)}-{uuid}{ext}"`, as required by the task description.